share/completions/eselect.fish

```diff
@@ -33,12 +34,19 @@ function __fish_complete_eselect_action_options
     __fish_eselect_cmd $cmdl[2..-1] | sed -n -re $findregexp | __fish_sgrep '^    --' | sed -re $parseregexp
 end
 
+function __fish_complete_eselect_php_actions
+    set -l sedregexp 's/^  .([0-9]+).   ([A-Za-z0-9\.]+).?+/\1\t\2/'
+
+    if test (__fish_print_cmd_args_without_options)[3] = 'set'
+        eselect php list (__fish_print_cmd_args_without_options)[-1] | sed -r $sedregexp
```

Comment:
I didn't manage to build fish from git, it just breaks. But I'll include some sample output from `eselect` below.

```
root@trillian ~# eselect php list cli
  [1]   php5.6 *
  [2]   php7.0
```

The one with the star is the selected version.

I just put a commit in with sed since I can't get the git version to behave so I can't test it.